docs/stake-pool-course/handbook/grafana-dashboard-tutorial.md

```diff
@@ -0,0 +1,492 @@
+---
+id: grafana-dashboard-tutorial
+title: Grafana Dashboard Tutorial
+sidebar_label: Grafana Dashboard Tutorial
+description: "Stake pool course: Grafana Dashboard Tutorial"
+image: ./img/og-developer-portal.png
+---
+
+Once you have sucessfully set-up a Cardano pool, then comes the most beautifull part - setting up your Dashboard and Alerts! You surely dont want to gaze the PC whole day, or?
```

Comment:
Great work @sandy4de. I'd recommend to remove "You surely dont want to gaze the PC whole day, or?" unless of course you have a strong opinion to keep it. :)) I think it's a little too "funny".

Reply:
I agree :)